### PR TITLE
set legalForm of Participant to optional

### LINKED ISSUE #534: Participant.legalForm is mandatory but should be optional
https://github.com/International-Data-Spaces-Association/InformationModel/blob/018abfe8a1e96ac842cccc99d4ae1dddfdc838cc/testing/participant/ParticipantShape.ttl#L110 states that Participant.legalForm has cardinality min 1 and max 1 --> is mandatory. However, there are many use cases where I want to speak about an IDS Participant without even knowing his legalForm. Therefore, I am voting to remove the min 1 line.

### PR DESCRIPTION
**Pull request by changqin26**
This resolves #534.

### CODE REVIEW

File: testing/participant/ParticipantShape.ttl [hunk 1]
```diff
@@ -106,9 +106,9 @@ shapes:ParticipantShape
 		sh:datatype xsd:string ;
 		sh:severity sh:Violation ;
 		#TODO for major release: Enforce property
-		sh:minCount 1 ;
+		sh:minCount 0 ;
 		sh:maxCount 1 ;
-		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): An idsParticipant must have exactly one legal form as xsd:string linked through the ids:legalForm property."@en ;
+		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): The legal form for an idsParticipant is optional."@en ;
```
Comment (JohannesLipp) [inline testing/participant/ParticipantShape.ttl:111]:
```suggestion
		sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/participant/ParticipantShape.ttl> (ParticipantShape): An idsParticipant must have zero or one legal form as xsd:string linked through the ids:legalForm property."@en ;
```

File: testing/participant/ParticipantShape.ttl [hunk 2]
```diff
@@ -106,9 +106,9 @@ shapes:ParticipantShape
 		sh:datatype xsd:string ;
 		sh:severity sh:Violation ;
 		#TODO for major release: Enforce property
```
Comment (JohannesLipp) [inline testing/participant/ParticipantShape.ttl:108]: remove TODO item?

**Review by JohannesLipp — CHANGES_REQUESTED** [review]

File: testing/participant/ParticipantShape.ttl (hunk 2 above)
Comment (changqin26) [reply testing/participant/ParticipantShape.ttl]: Done

File: testing/participant/ParticipantShape.ttl (hunk 1 above)
Comment (changqin26) [reply testing/participant/ParticipantShape.ttl]: Done

**Review by JohannesLipp — APPROVED** [review]

**Pull request merged**